include/OpenSpaceToolkit/Physics/Units/Mass.hpp

```diff
@@ -80,7 +80,7 @@ class Mass : public Unit
 
         static Mass             Kilograms                                   (   const   Real&                       aValue                                      ) ;
 
-        static Mass             Parse                                       (   const   String&                     aString                                     ) ;
+        // static Mass             Parse                                       (   const   String&                     aString                                     ) ;
```

Comment:
??

Reply:
It is not implemented in the cpp for some reason, so I commented it out in here as well, otherwise it can break the python binding with an underfined reference error

Reply:
Another possibility would be to simply add `throw ostk::core::error::runtime::ToBeImplemented("Mass :: Parse") ;` to the `.cpp` for now?